en/user-guide.rst

```diff
@@ -57,13 +57,19 @@ You can set the CRS you want to use for geometry tables defined in your Interlis
 **5.	Advanced options (only for Interlis mode)**
 Since Project Generator uses ili2db when in Interlis mode, you can set advanced options that determine how your conceptual object-oriented model is mapped into a PostgreSQL/PostGIS  relational model.
 
+***Inheritance***
+
 ili2db allows you to use three types of inheritance mapping: "noSmartMapping", "smart1inheritance" and "smart2inheritance":
 
  + **noSmartMapping** This kind of inheritance maps all classes from the source Interlis model into the target database schema. Each class mantains all its attributes.
 
  + **smart1inheritance** This kind of inheritance prefers to create parent classes into the database, as long as they fulfill some specific requirements. Attributes from children classes will not be lost, since they are transferred into parent ones.
 
  + **smart2inheritance** This kind of inheritance prefers to create children classes into the database. Attribute from parent classes will not be lost, since they are transferred into children ones.
+ 
+***Extra Meta Attributes File***
+ 
+A \*.toml file containing values for meta attributes (like dispExpression) instead of having them directly in the Interlis file. The file can be selected per module configuration.
```

Comment:
module -> model?

Reply:
Arg...

Reply:
I think the (like dispExpression) can be removed here in favor of a new chapter about meta attributes, extra meta attributes (toml file) and project generator specific meta attributes.

Reply:
Okay. So should I write a new chapter about these things? 
I was not sure how deep I should go into this, to keep the documentation slim.

New chapter *Meta Atttributes*  on the end of this document? 
Or a link to a new document?

Reply:
I'd put this in a new document.
The paragraph written here is perfectly fine for the everyday projectgenerator user, while the other information is in-depth for the advanced interlis user.

Reply:
Okay. I will do that...